src/js/background/backgroundLogic.js

```diff
@@ -343,7 +350,13 @@ const backgroundLogic = {
     let pos = 0;
 
     // Let's collect UCIs/tabs for this window.
+    /** @type {Map<string, {order: string, tabs: Tab[]}>} */
     const map = new Map;
+    /** @type {boolean} */
+    let lastTabIsInTabGroup = tabs.length > 0
+      && tabs[tabs.length - 1].groupId
+      && tabs[tabs.length - 1].groupId !== TAB_GROUP_ID_NONE;
```

Comment:
Although in practice `0` is not used as a value, in theory 0 could also be a valid groupId.

All tab groups have non-negative IDs, and `undefined >= 0` is always false, so you can simplify this logic to:

```suggestion
    let lastTabIsInTabGroup = tabs.at(-1)?.groupId >= 0;
```

Reply:
Looks like the eslint configuration for this project uses the ECMAScript 2018 parser but the optional chaining operator was specified in ECMAScript 2020. I'm going to use `Array.at` and check the result instead of using `?.`. I'll open a separate PR to bump up the parser level of the eslint config.